be/src/exec/vectorized/olap_meta_scan_node.cpp

```diff
@@ -9,7 +17,9 @@ OlapMetaScanNode::OlapMetaScanNode(ObjectPool* pool, const TPlanNode& tnode, con
           _is_init(false),
           _tuple_id(tnode.olap_scan_node.tuple_id),
           _meta_scan_node(tnode.meta_scan_node),
-          _desc_tbl(descs) {}
+          _desc_tbl(descs) {
+    _name = "olap_meta_scan";
+}
```

Comment:
_name("olap_meta_scan") {
}

Reply:
> _name("olap_meta_scan") { }

_name is a member of the parent class `ScanNode`. It seems that we can't put it in the initialization list.